### ISSUE TITLE
Team Section Social Icons Causing WCAG 2.0 2.4.4 To Be Out of Compliance

### ISSUE THREAD

**Issue by user4394** [Codeinwp/themeisle-companion#748]
Within this file: obfx_modules/companion-legacy/inc/hestia/inc/sections/hestia-team-section.php

The social media icons are missing either alt text or Aria Labels, which makes my site out of compliance for WCAG 2.0 2.4.4 compliance for accessibility requirements.

> 2.4.4 Link Purpose (In Context): The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, except where the purpose of the link would be ambiguous to users in general. (Level A)

Can this issue be fixed? Maybe something like? Or by pulling in the icon's name or by adding another field for the label to the Customizer in WordPress
`$link .= ' class="btn btn-just-icon btn-simple" aria-label="Social Media and Contact Links"><i class="' . esc_attr( hestia_display_fa_icon( $social_icon ) ) . '"></i></a>';
`

Thank you!

**Comment by girishpanchal30** [comment]
Added [aria-label](https://tinyurl.com/2hxtnr3j) text in this PR https://github.com/Codeinwp/themeisle-companion/pull/772, please check with the build zip

**Comment by user4394** [comment]
Thank you for fixing this! When can I expect to see the updated Wordpress Plugin auto-update to include the fix?
